lesson_09/oop/oop_app/src/test/java/com/codedifferently/lesson9/mohamedibrahim/LibraryTest.java

```diff
@@ -0,0 +1,36 @@
+package com.codedifferently.lesson9.mohamedibrahim;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.ArrayList;
+import org.junit.jupiter.api.Test;
+
+class LibraryTest {
```

Comment:
Should have more tests than this given the size and scope of your `Library` implementation. Each method should have at least one test.